theme.config.tsx

```diff
@@ -235,7 +75,29 @@ export default {
     defaultMenuCollapseLevel: 1,
   },
   footer: {
-    content: Footer,
+    component: () => (
+      <Footer
+        extraLinks={[
+          {
+            title: "Community Grant",
+            route: "https://graphql.org/foundation/community-grant/",
+          },
+          {
+            title: "Code of Conduct",
+            route: "https://graphql.org/codeofconduct/",
+          },
+          {
+            title: "Brand",
+            route: "https://graphql.org/brand/",
+          },
+          // {
+          //   title: "Swag Shop",
+          //   route: "https://store.graphql.org/",
+          // },
```

Comment:
@saihaj the footer in the design has links for the future state, with some pages that we don't have up yet, right?

So we probably wanna be somewhere in between current and designed.

The current footer includes a swag shop link. Should we get rid of the `/spec` link in favor of this one?